docs/compilers/CSharp/Compiler Breaking Changes - VS2019.md

```diff
@@ -0,0 +1,10 @@
+**This document lists known breaking changes in Roslyn 3.0 (Visual Studio 2019) from Roslyn 2.* (Visual Studio 2017)
+
+*Breaks are formatted with a monotonically increasing numbered list to allow them to referenced via shorthand (i.e., "known break #1").
+Each entry should include a short description of the break, followed by either a link to the issue describing the full details of the break or the full details of the break inline.*
+
+1. Previously, we allowed the `Microsoft.CodeAnalysis.EmbeddedAttribute` to be referenced from a source module.
```

Comment:
>to be referenced from [](start = 73, length = 21)

declared in source?

Reply:
No, it can still be declared in source. The scenario that changed  is with a module added that declares that type.

---
In reply to: [216033962](https://github.com/dotnet/roslyn/pull/29180#discussion_r216033962) [](ancestors = 216033962)

Reply:
> No, it can still be declared in source. The scenario that changed  is with a module added that declares that type.

So this is not about a **source** module, this is a bout an added module. Also consider the other comment about using "referenced" term. 

---
In reply to: [216041844](https://github.com/dotnet/roslyn/pull/29180#discussion_r216041844) [](ancestors = 216041844,216033962)